CLI/test/CLITest.cs

```diff
@@ -17,7 +18,7 @@ public class CLITest {
             //Check that the actual "output" folder (the one created by the CLI) match the content of the expected "output" folder
             //the one on Git.
             //The number of files and the content of the files must be identical
-            CLITestHelper.Test("parse_1", ReturnCode.Success);
+            CLITestHelper.Test("parse_1", ReturnCode.OutputFileMissing);   
```

Comment:
Our current tests lack a proper description. You cannot guess the purpose of each test easily.


This test was about parsing a file and see that no errors was found. No output file was expected because no generation was asked.
The purpose of the test is now to check that a mandatory argument is missing.


 - Can you keep this simple parsing test (parse_1) and create a new one for the missing argument?
- Can you push a description/purpose of each test from your point of view? I'll complete the description if needed.